lib/brakeman.rb

```diff
@@ -180,6 +180,8 @@ def self.get_formats_from_output_format output_format
       [:to_json]
     when :markdown, :to_markdown
       [:to_markdown]
+    when :cc, :codeclimate, :to_codeclimate
```

Comment:
Do you want people to be able to do `brakeman -o report.cc`? If so it should be added below in `get_formats_from_output_files`.